From 0c3a8dea88a7c7f3fdda5000d3e77b61bfbe7f6a Mon Sep 17 00:00:00 2001
From: WANG Cong <xiyou.wangcong@gmail.com>
Date: Fri, 7 Oct 2011 15:44:10 +0800
Subject: [PATCH] update the documentation of '--[no]prefix'

In

commit fd786adcf515d9d3ee77eb29fa4c6b60020c7209
Author: Harald Hoyer <harald@redhat.com>
Date:   Wed Apr 20 16:47:40 2011 +0200

    dracut: make prefix configurable

Harald changed the meaning of --prefix/--noprefix, but
forgot to update their documentation. This patch
fixes that.

Signed-off-by: WANG Cong <xiyou.wangcong@gmail.com>
---
 dracut.8.xml |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/dracut.8.xml b/dracut.8.xml
index 11ea8c3..709f1a6 100644
--- a/dracut.8.xml
+++ b/dracut.8.xml
@@ -245,10 +245,10 @@ include in the generic initramfs. This parameter can be specified multiple times
         </varlistentry>
         <varlistentry>
           <term>
-            <option>--prefix</option>
+            <option>--prefix&nbsp;<replaceable>&lt;dir&gt;</replaceable></option>
           </term>
           <listitem>
-            <para>prefix initramfs files with /run/initramfs/</para>
+            <para>prefix initramfs files with the specified directory</para>
           </listitem>
         </varlistentry>
         <varlistentry>
@@ -256,7 +256,7 @@ include in the generic initramfs. This parameter can be specified multiple times
             <option>--noprefix</option>
           </term>
           <listitem>
-            <para>do not prefix initramfs files with /run/initramfs/ (default)</para>
+            <para>do not prefix initramfs files (default)</para>
           </listitem>
         </varlistentry>
         <varlistentry>
